test/unit/reductions/exponentiated_gradient/test_bgl_GPA.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LinearRegression
+from fairlearn.metrics import group_mean_prediction, group_mean_squared_error, group_mean_overprediction
+from fairlearn.reductions import GridSearch, DemographicParity, EqualizedOdds, GroupLossMoment, \
+    ZeroOneLoss, SquareLoss
+
+
+def main():
```

Comment:
Could this be rewritten as a `pytest` test, please? That way we can run it in our test harness.

Reply:
yes agreed-- I'll fix it and add it as an additional test to the test_grid_search_arguments.py file that already has some BGL pytests

Reply:
yes agreed-- fixed it and added it as an additional test to the test_grid_search_arguments.py file that already has BGL pytests